mne/utils.py

```diff
@@ -47,6 +47,8 @@
 
 from .fixes import _get_args
 
+from mne import __version__
```

Comment:
avoid absolute imports, instead in the function that needs it nest a `from . import __version__`